src/locales/en-AU.json

```diff
@@ -2,8 +2,8 @@
   "mnoe_admin_panel.constants.user_roles.member": "Member",
```

Comment:
The user_roles needs to be renamed to "Admin" and "Manager"

Reply:
renamed with ea8557c, the selected line does not need changes